.github/workflows/dependabot-merger.yml

```diff
@@ -0,0 +1,128 @@
+name: Merge Dependabot PRs
+on:
+  schedule:
+    - cron: "0 9 * * 1" # Run this workflow every Monday at 9:00
```

Comment:
And if you can assign it to the current on-call, maybe run the job at whatever time the on-call bot updates + some delay.